app/src/components/intelligence/memory-workspace.css

```
@@ -0,0 +1,613 @@
+  font-weight: 500;
+}
+
+.mw-mentioned-table {
+  width: 100%;
+  border-collapse: collapse;
+}
+.mw-mentioned-row {
+  cursor: pointer;
+  background: transparent;
+  border: none;
+  padding: 0;
```

Stylelint (17.10.0)

[error] Duplicate property "padding" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)